resilience4j-framework-common/src/main/java/io/github/resilience4j/common/retry/configuration/RetryConfigurationProperties.java

```diff
@@ -243,6 +245,12 @@ private void configureRandomizedWait(InstanceProperties properties, RetryConfig.
         @Nullable
         private Duration waitDuration;
 
+        /*
+         * retry intervalBiFunction class to be used to calculate wait based on exception or result
+         */
+        @Nullable
+        private Class<? extends IntervalBiFunction<Object>> intervalBiFunction;
```

Comment:
@RobWin Should I throw and exception when both `RetryConfigurationProperties.InstanceProperties.waitDuration` and `RetryConfigurationProperties.InstanceProperties.intervalBiFunction` are used or let `RetryConfig.Builder.build` to do the job?

Reply:
Let RetryConfig.Builder.build do the job